.github/workflows/install-ssf.yaml

```
@@ -0,0 +1,18 @@
+name: CI
+on:
+  - pull_request
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    name: build example and deploy to minikube
+    steps:
+    - uses: actions/checkout@v2
+    - name: Start minikube
+      uses: medyagh/setup-minikube@master
+    - name: Try the cluster !
+      run: kubectl get pods -A
+    - name: Deploy SSF to minikube
+      run:
```

A pipe here is most likely needed.

```suggestion
      run: |
```